### PR TITLE
Add documentation for persistent VM instance snapshots functionality

### PR DESCRIPTION
**Pull request by claudiolor**
# Description

This PR is aimed to provide the documentation about the persistent VM instance snapshots functionality.

### CODE REVIEW

**Review by giorio94 — APPROVED** [review]
/lgtm

File: operators/README.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ The Instance Operator implements the backend logic necessary to spawn new enviro
 
 - **Template** defines the size of the execution environment (e.g.; Virtual Machine), its base image and a description. This object is created by managers and read by users, while creating new instances.
 - **Instance** defines an instance of a certain template. The manipulation of those objects triggers the reconciliation logic in the operator, which creates/destroy associated resources (e.g.; Virtual Machines).
+- **InstanceSnapshot** defines a snapshot for a persistent VM instance, once the operator finds this resource, it will start the snapshot creation process.
```
Comment (frisso) [inline operators/README.md:79]:
```suggestion
- **InstanceSnapshot** defines a snapshot for a persistent VM instance. Once this resource is created, the associated operator will start the snapshot creation process.
```

File: operators/README.md [hunk 2]
```diff
@@ -76,6 +76,7 @@ The Instance Operator implements the backend logic necessary to spawn new enviro
 
 - **Template** defines the size of the execution environment (e.g.; Virtual Machine), its base image and a description. This object is created by managers and read by users, while creating new instances.
 - **Instance** defines an instance of a certain template. The manipulation of those objects triggers the reconciliation logic in the operator, which creates/destroy associated resources (e.g.; Virtual Machines).
+- **InstanceSnapshot** defines a snapshot for a persistent VM instance. Once this resource is created, the associated operator will start the snapshot creation process.
```
Comment (frisso) [inline operators/README.md:79]:
```suggestion
- **InstanceSnapshot** defines a snapshot for a persistent VM instance. The associated operator will start the snapshot creation process once this resource is created.
```

**Review by frisso — APPROVED** [review]

**Pull request merged**